libs/payments/ui/src/lib/client/components/Breadcrumbs/index.tsx

```diff
@@ -62,8 +66,46 @@ export function Breadcrumbs(args: {
   }
 
   return (
-    <nav className="p-4 tablet:p-6" aria-label="Breadcrumb">
-      <ol className="flex items-center">
+    <nav
+      className="flex items-center h-11 tablet:h-[76px] p-4 tablet:p-6 border-b tablet:border-b-0 border-grey-100"
+      aria-label="Breadcrumb"
+    >
+      <ol className="tablet:hidden">
+        {breadcrumbs.map(({ label, href }, i) => {
+          const isPrev = i === breadcrumbs.length - 2;
+          if (!isPrev) return null;
+
+          const url = new URL(href);
+          url.search = searchParams.toString();
+          href = url.href;
+
+          return (
+            <li key={`breadcrumb-nav-${i}`}>
+              <Link
+                href={href}
+                className="inline-flex items-center text-blue-600 gap-3"
+                aria-label={l10n.getString(
+                  'subscription-management-breadcrumb-back-aria',
+                  {
+                    page: label
+                  },
+                  `Go back to ${label}`
+                )}
+              >
+                <Image
+                  src={leftArrowIcon}
+                  alt=""
+                  aria-hidden="true"
+                  className="w-6 h-6"
+                />
+                <span>{label}</span>
```

Comment:
Nit: The label and new arrow icons don't appear centered even though `items-center` is used - the font itself is a bit low. Can we add some padding beneath the label (I think `pb-0.5` should be enough)? 

We should also match the label/text behavior where it is underlined when a user hovers over it, similar to the breadcrumbs in tablet/desktop view.